cardinal/server/handler/debug.go

```diff
@@ -0,0 +1,66 @@
+package handler
+
+import (
+	"encoding/json"
+
+	"github.com/gofiber/fiber/v2"
+
+	"pkg.world.dev/world-engine/cardinal/search/filter"
+	servertypes "pkg.world.dev/world-engine/cardinal/server/types"
+	"pkg.world.dev/world-engine/cardinal/types"
+)
+
+type DebugStateRequest struct{}
+
+type debugStateElement struct {
+	ID         types.EntityID             `json:"id"`
+	Components map[string]json.RawMessage `json:"components" swaggertype:"object"`
+}
+
+type DebugStateResponse []*debugStateElement
```

Comment:
is there a reason why this is a pointer to struct? without pointer seems cleaner.

Reply:
No reason. It was copied from the plugin code. Changed to value.